libraries/node-core-library/src/LockFile.ts

```diff
@@ -6,28 +6,105 @@ import * as path from 'path';
 import * as child_process from 'child_process';
 import { setTimeout } from 'timers';
 
+/**
+ * Parses the process start time from a linux /proc/1/stat file.
+ * @param stat The contents of a linux /proc/1/stat file.
+ * @returns The process start time in yiffies.
+ */
+export function getProcessStartTimeFromProcStat (stat: string): string | undefined {
+  // Parse value 22.
```

Comment:
Could you define 22 as a local constant in this file, and cite this text from the Linux manpage?

```
(22) starttime  %llu
The time the process started after system boot. In kernels before Linux 2.6, this value was expressed in jiffies. Since Linux 2.6, the value is expressed in clock ticks (divide by sysconf(_SC_CLK_TCK)).
The format for this field was %lu before Linux 2.6.
```